__tests__/app.test.js

```diff
@@ -1,9 +1,12 @@
+const request = require("supertest");
+const app = require("../app");
 const endpointsJson = require("../endpoints.json");
+
 /* Set up your test imports here */
 
 /* Set up your beforeEach & afterAll functions here */
 
-describe("GET /api", () => {
+describe.only("GET /api", () => {
```

Comment:
Please remove all .skip's and .only's from tests before pushing your work. From now on, we'll have to ask you to resubmit if we see any of those. We can't be sure all your tests are passing otherwise.